tests/e2e-instrumentation/instrumentation-apache-httpd/chainsaw-test.yaml

```diff
@@ -4,14 +4,25 @@ kind: Test
 metadata:
   name: instrumentation-apache-httpd
 spec:
-  namespaceTemplate:
```

Comment:
This was changed as when the annotation is applied by template, the annotation openshift.io/sa.scc.mcs is missing. 

`chainsaw-becoming-lab": unable to find annotation openshift.io/sa.scc.mcs`

When applied by kubectl, the annotation with the right value is added.

Reply:
Things behave differently when the ns is annotated than when the namespace is created with the annotation already set ?

Reply:
Yes, with the CLI the annotation annotation openshift.io/sa.scc.mcs with the correct value is added.

Reply:
I don't know much about openshift but it sounds strange to me that things behave differently when the annotation is added afterward 🤷

Reply:
For the time being we can get this merged so that the tests do not fail on OpenShift. I'm consulting with our OpenShift team on this and will try to find a way to get it working with the template.

Reply:
Sure, i was just curious :)